test/models/content_test.rb

```diff
@@ -20,5 +20,36 @@ def test_update_repo_name
 
       assert_includes repo.reload.name, "FOOBAR"
     end
+
+    def test_import_all
```

Comment:
@parthaa good call. here's a test

Reply:
Not a great test - but the ProductContentImporter is where the heavy lifting happens and it's well tested